modules/tensor_mechanics/src/actions/PressureActionTM.C

```diff
@@ -0,0 +1,102 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+#include "PressureActionTM.h"
+
+#include "Factory.h"
+#include "FEProblem.h"
+#include "Parser.h"
+
+template<>
+InputParameters validParams<PressureActionTM>()
+{
+  InputParameters params = validParams<Action>();
+  params.addRequiredParam<std::vector<BoundaryName> >("boundary", "The list of boundary IDs from the mesh where the pressure will be applied");
+  params.addRequiredParam<NonlinearVariableName>("disp_x", "The x displacement");
+  params.addParam<NonlinearVariableName>("disp_y", "", "The y displacement");
+  params.addParam<NonlinearVariableName>("disp_z", "", "The z displacement");
+
+  params.addParam<std::vector<AuxVariableName> >("save_in_disp_x", "The save_in variables for x displacement");
+  params.addParam<std::vector<AuxVariableName> >("save_in_disp_y", "The save_in variables for y displacement");
+  params.addParam<std::vector<AuxVariableName> >("save_in_disp_z", "The save_in variables for z displacement");
+
+  params.addParam<Real>("factor", 1.0, "The factor to use in computing the pressure");
+  params.addParam<FunctionName>("function", "The function that describes the pressure");
+  params.addParam<PostprocessorName>("postprocessor", "", "The postprocessor that describes the pressure");
+  return params;
+}
+
+PressureActionTM::PressureActionTM(const std::string & name, InputParameters params) :
+  Action(name, params),
+  _boundary(getParam<std::vector<BoundaryName> >("boundary")),
+  _disp_x(getParam<NonlinearVariableName>("disp_x")),
+  _disp_y(getParam<NonlinearVariableName>("disp_y")),
+  _disp_z(getParam<NonlinearVariableName>("disp_z")),
+  _factor(getParam<Real>("factor")),
+  _postprocessor(getParam<PostprocessorName>("postprocessor")),
+  _kernel_name("PressureTM"),
+  _use_displaced_mesh(true)
+{
+  _save_in_vars.push_back(getParam<std::vector<AuxVariableName> >("save_in_disp_x"));
+  _save_in_vars.push_back(getParam<std::vector<AuxVariableName> >("save_in_disp_y"));
+  _save_in_vars.push_back(getParam<std::vector<AuxVariableName> >("save_in_disp_z"));
+
+  _has_save_in_vars.push_back(params.isParamValid("save_in_disp_x"));
+  _has_save_in_vars.push_back(params.isParamValid("save_in_disp_y"));
+  _has_save_in_vars.push_back(params.isParamValid("save_in_disp_z"));
+}
+
+void
+PressureActionTM::act()
+{
+  // Determine number of dimensions
+  unsigned int dim(1);
+  if (_disp_y != "")
+  {
+    ++dim;
+  }
```

Comment:
We generally prefer not to use braces when there's only a single statement in a branch. Much of the code in the modules has this but we are trying to move away from that.